packages/camera/camera_windows/windows/record_handler.cpp

```diff
@@ -160,6 +176,14 @@ HRESULT RecordHandler::InitRecordSink(IMFCaptureEngine* capture_engine,
     return hr;
   }
 
+  if (0 < fps_) {
```

Comment:
These should be comparing directly against the special value, not use a different comparison.

Reply:
refactored using `std::optional::has_value`